lib/ts/recipe/passwordless/components/themes/signInUp/phoneNumberInput.tsx

```diff
@@ -71,10 +71,7 @@ function PhoneNumberInput({
     const handleChange = useCallback(
         (newValue: string) => {
             if (onChangeRef.current !== undefined) {
-                onChangeRef.current({
-                    id: name,
-                    value: newValue,
-                });
+                onChangeRef.current(newValue);
```

Comment:
Is this change detected cause of a TS error or something else?

Reply:
yes